engine/src/test/java/io/camunda/zeebe/engine/perf/EngineLargeStatePerformanceTest.java

```diff
@@ -135,12 +135,12 @@ public Record<?> measureProcessExecutionTime() {
   void shouldProcessWithinExpectedDeviation(final JMHTestCase testCase) {
     // given - an expected ops/s score, as measured in CI
     // when running this test locally, you're likely to have a different score
-    final var referenceScore = 450;
+    final var referenceScore = 1000;
 
     // when
     final var assertResult = testCase.run();
 
     // then
-    assertResult.isWithinDeviation(referenceScore, 0.15);
+    assertResult.isAtLeast(referenceScore, 0.25);
```

Comment:
used this method to set a baseline, allowing higher gains without failing in future, previously introduced by @npepinpe .
This means we will miss new performance gains though but have less maintenance effort 😅  wdyt? @camunda/zeebe-distributed-platform

Reply:
🤷🏼 I guess I was the only one which wanted to have this. I would like to know when we impact in anyway the performance, but lets wait for the others.